oidc-consumer/src/index.ts

```diff
@@ -15,6 +15,7 @@ class OidcConsumer {
   callback_route?: string;
   default_callback_route?: string;
```

Comment:
are we using the default_callback_route anywhere?

Reply:
stale prop, removed @ [2f57441](https://github.com/getfundwave/idp-consumer/pull/2/commits/2f5744147af65baa1df39a472dbd8dc3f45842eb)